src/common/components/RangeAggregation.jsx

```diff
@@ -141,9 +141,9 @@ class RangeAggregation extends Component {
       max, min, data, endpoints,
     } = this.state;
     const sliderMarks = { [max]: max, [min]: min };
-    const { width, height, name } = this.props;
+    const { height, name } = this.props;
     return (
-      <div>
+      <div style={{ width: '100%' }}>
```

Comment:
can we avoid hardcoded styles? can we add, for the time being, a separated file for each component?

Reply:
actually we still haven't sorted it out how we will do styling. If we keep them in separate files selectors & class names should be unique since they will be global. And after if we decide to go for encapsulated styles, then we'll (optionally) update class names and selectors.

Reply:
https://github.com/paypal/glamorous

https://github.com/smyte/jsxstyle